docs/releases/4.1.txt

```diff
@@ -433,6 +433,8 @@ Miscellaneous
 * The ``size`` argument of the undocumented
   ``django.views.static.was_modified_since()`` function is removed.
 
+* The admin now logging out via ``POST`` requests.
```

Comment:
Needs at least a _does_ here, but does one _Do logging out_? — I'm not sure 

Maybe:

```suggestion
* The admin log out UI now uses ``POST`` requests.
```

Reply:
Updated :+1: